### ISSUE TITLE
The loss function does not converge

### ISSUE THREAD

**Issue by hyperion0819** [bubbliiiing/arcface-pytorch#6]
use my own datasets,however,the acc of train.py always equals 0.At the same time,the loss function does not converge.Why? i need adjust hyper-parameter(s or m)?

**Comment by hyperion0819** [comment]
> No, it needs lots of datasets. How many images do you have?

您好 我使用的是鱼体数据集 有30个类别 358张图片 但我将这些图片进行了复制 得到了500个文件夹 5000张图片（这些图片与原图片没有区别）这样可以吗

**Comment by bubbliiiing** [comment]
复制？？复制为什么可以到500哥文件夹？那同一个类在不同文件夹不就错了吗

**Comment by hyperion0819** [comment]
> 复制？？复制为什么可以到500哥文件夹？那同一个类在不同文件夹不就错了吗

嗯嗯 明白了 总之就是需要大量数据集 其次不用更改s m参数 训练的准确率就能提升是吗

**Comment by bubbliiiing** [comment]
s m参数是啥……我其实都不知道

**Comment by bubbliiiing** [comment]
head里面的么？不需要改

**Comment by hyperion0819** [comment]
> head里面的么？不需要改

好的 谢谢
